IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_RailwayBalise/DocPropertySet.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RailwayBalise" UniqueId="0de0be0f-d959-6b3c-ea26-3288507ff368" ApplicableType="IfcCommunicationsAppliance/TRANSPONDER" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="NominalHeight_1Fy4XYaJXDTBDDb9f06ofi" />
+		<DocProperty xsi:nil="true" href="NominalWidth_2YZfyYRY1Aa9bsWGvZEaNq" />
+		<DocProperty xsi:nil="true" href="NominalWeight_1NkoWVgmHA5xrENqUppTds" />
+		<DocProperty xsi:nil="true" href="NominalLength_07RHK0qUaHuO00025QrEV" />
+		<DocProperty xsi:nil="true" href="FailureInformation_30Y_YD1x1DY8_4MwqYT1kM" />
+		<DocProperty xsi:nil="true" href="RailwayBaliseType_0vpUHwHvz9kAU_7eMQONr9" />
+		<DocProperty xsi:nil="true" href="DetectionRange_18tiQnmqr3a9LRMZsdhle" />
+		<DocProperty xsi:nil="true" href="InformationLength_05Ecr739L7IvyGuZ7C2zv" />
+		<DocProperty xsi:nil="true" href="TransmissionRate_16aHHnGsH6whpVLYqdzOc" />
+		<DocProperty xsi:nil="true" href="OperationalTemperatureRange_2_eMO0qS0HuO00025QrEV" />
+		<DocProperty xsi:nil="true" href="IP_Code_1MhaW0qTyHuO00025QrEV" />
```

Comment:
Could this be handled through classification instead?

Reply:
marking does this need to be removed?